tests/validators/invalid_urls.txt

```diff
@@ -2,6 +2,8 @@ foo
 http://
 http://example
 http://example.
+http://example.com:-1
```

Comment:
This raises a `ValidationError` without the patch, please move it to a separate commit.